core/src/main/java/com/distkv/dst/core/struct/slist/SortedListLinkedListImpl.java

```diff
@@ -0,0 +1,261 @@
+package com.distkv.dst.core.struct.slist;
+
+import com.distkv.dst.common.entity.sortedList.SortedListEntity;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * The linkedlist implementation of the Dst SortedList.
+ */
+public final class SortedListLinkedListImpl
```

Comment:
I'd like to name this as SortedListLinkedImpl

Reply:
I also would like to rename this. And what about **SortedLinkedListImpl**?

Reply:
I still prefer SortedListLinkedListImpl because this is a `SortedList` and impled in `Linked`.